templates/layout.html

```diff
@@ -81,13 +83,5 @@
         <small>© OpenStreetMap US | <a href="mailto:membership@openstreetmap.us">membership@openstreetmap.us</a></small>
     </div>
 
-  <script type="text/javascript">
```

Comment:
So this code fetches the svg sprite and stuffs it somewhere into the DOM so that the `<use>` can refer to the symbol.  We don't need it if the logo svg is already inlined into the DOM.